cmake/compilers/IntelLLVM.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2023 Intel Corporation
+# Copyright (c) 2021-2023 Intel Corporation
```

Comment:
What is the purpose of the 2020&rarr;2021 change?

Reply:
@bavirnen you shouldn't update the first year in copyright, only the last one.

Reply:
sorry misunderstood this statment. **_Since you edited this file, please update copyright year from 2021 to 2023._**. let me correct it.

Reply:
updated the suggested change